packages/front-end/components/Experiment/CompactResults.tsx

```diff
@@ -535,6 +579,11 @@ const CompactResults: FC<{
                             </em>
                           </small>
                         </div>
+                        {highVariance && (
+                          <div className="badge badge-pill badge-warning">
+                            Suspiciously high variance
+                          </div>
+                        )}
```

Comment:
I don't think this is the right place for this warning.  If the percent change is past the threshold, we should treat it like a "not enough data" warning - hide the chance to beat baseline, risk, and percent change columns and just show the warning instead since those stats won't tell you anything useful.  If a variation is up > 50%, the risk is always going to be 0, the chance to beat control is always going to be 100%, and the percent change graph is going to be really skewed and mess up the other metrics.

![image](https://user-images.githubusercontent.com/1087514/131692320-a57071a2-3252-4245-a406-5299ce24cd01.png)

This might not be the perfect visual treatment, but something like that.